build/templates/installation.inc.mako

```diff
@@ -0,0 +1,28 @@
+<%
+    import build.helper as helper
+
+    config        = template_parameters['metadata'].config
+    attributes    = helper.filter_codegen_attributes(config['attributes'])
+    functions     = helper.filter_codegen_functions(config['functions'])
+    module_name   = config['module_name']
+    driver_name   = config['driver_name']
+    c_function_prefix = config['c_function_prefix']
+%>\
+
+.. _${module_name}_installation-section:
+
+Installation
```

Comment:
I like having per driver top level readme.
I think the nimi-python project readme shouldn't have installation instructions. Seems overly repetitive.

Reply:
Add links from project readme to read the docs installation procedure per driver